resources/scripts/pytest_otel/README.md

```diff
@@ -0,0 +1,65 @@
+
+Features
+--------
+
+pytest-otel plugin for report APM traces about test executed.
+
+
+Requirements
+------------
+
+* opentelemetry-api == 1.2
+* opentelemetry-exporter-otlp == 1.2.0
+* opentelemetry-sdk == 1.2.0
+
+
+Installation
+------------
+
+You can install "pytest-otel" via `pip` or using the `setup.py` script.
+
+```
+git checkout https://github.com/elastic/apm-pipeline-library
+cd apm-pipeline-library/resources/scripts
+pip install ./pytest_otel
+```
+
+Usage
+-----
+
+pytest_otel is configured by adding some parameters to the pytest command line here are the descriptions
+
+* --endpoint: URL for the OpenTelemetry server. (Required). Env variable `OTEL_EXPORTER_OTLP_ENDPOINT`
+* --headers: Additional headers to send (i.e.: key1=value1,key2=value2). Env variable `OTEL_EXPORTER_OTLP_HEADERS`
+* --service-name: Name of the service. Env variable `OTEL_SERVICE_NAME`
+* --session-name: Name for the Main span reported.
+* --trace_id: Trace parent id. Env variable `TRACE_ID` see https://www.w3.org/TR/trace-context-1/#trace-context-http-headers-format
```

Comment:
Did you consider supporting consuming `traceparent` as an alternative to `trace_id`?

Reply:
the content is `traceparent` as it is specified at https://www.w3.org/TR/trace-context-1/#trace-context-http-headers-format. I do not really found documentation about `TRACE_ID` environment variable.

Reply:
I'm wondering if consuming a `traceparent` rather than `trace_id` wouldn't be better because what we need to connect a span to a parent span is not only the parent trace_id but also the parent span_id and the `trace_flags` to get the sampling decision `sampled`.
Note that the [`otel_cli`](https://github.com/packethost/otel-cli) typically consumes the `TRACEPARENT` environment header

Reply:
I guess I have to rename the flag because is not acurate

Reply:
```suggestion
* --traceparent: Trace parent id. Env variable `TRACEPARENT` see https://www.w3.org/TR/trace-context-1/#trace-context-http-headers-format
```